File: Training/URI/categoria-Iniciante/C/1179.c
#include<stdio.h>
#include<stdlib.h>

main() {
	int vetorPar[5];
	int vetorImpar[5];
	int n,i,j;
	int countPar = 0;
	int countImpar = 0;
	for(i = 0; i < 15; i++) {
		scanf("%d",&n);

		if(n % 2 == 0) {
			vetorPar[countPar] = n;
			countPar++;
		} else {
			vetorImpar[countImpar] = n;
			countImpar++;
		}

		if(countPar == 5) {
			for(j = 0; j < 5; j++) {
				printf("par[%d] = %d\n",j, vetorPar[j]);
			}

			countPar = 0;
		}

		if(countImpar == 5) {
			for(j = 0; j < 5; j++) {
				printf("impar[%d] = %d\n",j, vetorImpar[j]);
				countImpar = 0;
			}

		}
	}

	for(i = 0; i < countImpar; i++) {
		printf("impar[%d] = %d\n", i, vetorImpar[i]);
	}

	for(i = 0; i < countPar; i++) {
		printf("par[%d] = %d\n", i, vetorPar[i]);
	}
}

File: Training/URI/categoria-Iniciante/java/1179.java

import java.util.Scanner;;

public class Main {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int[] vetorPar = new int[5];
		int[] vetorImpar = new int[5];

		int n;
		int countPar = 0;
		int countImpar = 0;

		for(int i = 1; i <= 15; i++) {
			n = scanner.nextInt();

			if(n % 2 == 0) {
				vetorPar[countPar] = n;
				countPar++;
			} else {
				vetorImpar[countImpar] = n;
				countImpar++;
			}

			if(countPar == 5) {
				for(int j = 0; j < 5; j++) {
					System.out.println("par[" + j + "] = " + vetorPar[j]);
				}
				countPar = 0;
			}

			if(countImpar == 5) {
				for(int j = 0; j < 5; j++) {
					System.out.println("impar[" + j + "] = " + vetorImpar[j]);
				}
				countImpar = 0;
			}


		}

		for(int i = 0; i < countImpar; i++) {
			System.out.println("impar[" + i + "] = " + vetorImpar[i]);
		}

		for(int i = 0; i < countPar; i++) {
			System.out.println("par[" + i + "] = " + vetorPar[i]);
		}
	}
}
